backend/apiviews/author_views.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth.models import User
+
+from rest_framework import viewsets
+from rest_framework import mixins
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework import permissions
+from rest_framework.decorators import permission_classes
+
+from backend.serializers import UserSerializer, FriendSerializer
+from backend.models import User, Friend
+from backend.permissions import *
+
+from django.http import Http404
+from django.db.models import Q
+
+class AuthorViewSet(viewsets.ViewSet):
+    
+    def get_authors(self, request, *args, **kwargs):
+        '''
+        Get all the authors
+        '''
+        author = User.objects.all()
+        serializer = UserSerializer(author,many=True)
+        return Response(serializer.data)
+
+
+    def get_object(self, pk):
```

Comment:
No need to include this function as Viewset comes with this function by default already